heidrun/cdl.rb

```diff
@@ -12,9 +12,7 @@
   md5_hash = r['reference_image_md5'].first ? r['reference_image_md5'].first.value : nil
   image_url = nil
 
-  if !md5_hash.nil?
-    image_url = base_url + '150x150/' + md5_hash
-  end
+  image_url = base_url + '150x150/' + md5_hash unless md5_hash.nil?
```

Comment:
Could this whole block all be flattened:

```ruby
md5_hash = r['reference_image_md5'].first && r['reference_image_md5'].first.value
md5_hash && ("https://thumbnails.calisphere.org/clip/150x150/#{md5_hash}")
```

I'm okay with expanding this out for readability as desired, but the current version feels overly complex to me.